pkg/api/v1/register.go

```diff
@@ -86,6 +86,10 @@ func addKnownTypes() {
 		&ThirdPartyResourceList{},
 		&ThirdPartyResourceData{},
 	)
+
+	// Add common types
+	api.Scheme.AddKnownTypes("v1", &unversioned.Status{})
```

Comment:
Hm, I think this is OK. But maybe Scheme's AddKnownTypeWithName could help? I guess if we can make the swagger output exactly like it was before, that would be ideal.